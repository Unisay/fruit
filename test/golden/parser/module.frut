module Hello.Wonderful.World

exports -- (export1, export2, export3)
  - export1
  - export2
  - export3

imports 
  - Example.Import0
  - Example.Import1
    - foo
  - Example.Import2
    - bar
    - baz
  - Example.Import3 (f1, f2, f3)
