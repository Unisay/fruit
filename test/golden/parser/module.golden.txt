Right (SourceFile (Module ("Hello" :| [ "Wonderful" , "World" ])))